src/components/widget/Widget.js

```diff
@@ -0,0 +1,46 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import WidgetBody from './WidgetBody';
+import WidgetFooter from './WidgetFooter';
+import WidgetHeader from './WidgetHeader';
+import theme from './theme.css';
+
+const SIZES = {
+  small: 3,
+  medium: 4,
+  large: 5,
+};
+
+class Widget extends PureComponent {
+  render() {
+    const { children, size, ...others } = this.props;
+
+    return (
+      <Box className={theme['widget']} {...others}>
```

Comment:
We should also provide the possibility to add an additional class name (with `cx`).